subsys/net/ip/tcp_internal.h

```diff
@@ -367,6 +371,57 @@ void net_tcp_init(void);
 #define net_tcp_init(...)
 #endif
 
+/**
+ * @brief Set tcp specific options of a socket
+ *
+ * @param context Network context
+ *
+ * @return 0 on success, -EINVAL if the value is not allowed
+ */
+#if defined(CONFIG_NET_NATIVE_TCP)
+int net_tcp_set_option(struct net_context *context,
+			   enum tcp_conn_option option,
```

Comment:
And in this file as well.

Reply:
When I do so it looks like I cat a line length warning of over 100 characters, although the lines do not exceed 70 characters. Do you know what is wrong?

Reply:
I bet our editor is configured to use 4 spaces/tab, Zephyr uses 8
![image](https://user-images.githubusercontent.com/22167799/172785880-0a23d4b8-4ccc-4a69-a074-430e455f1e65.png)

Reply:
That explains, issue fixed